### DISCUSSION TITLE
exec /root/.start-container.sh: no such file or directory

### DISCUSSION THREAD

**Discussion by verovaleros** [stratosphereips/stratocyberlab#23] [Q&A] [answered]
When trying to start the lab after the update I'm getting 

```hackerlab | exec /root/.[start-container.sh](http://start-container.sh/): no such file or directory```

Is there a problem with the docker?

**Comment by verovaleros** [comment, marked as answer]
Answer from @LukasForst and @HappyStoic:

For people with Windows machines running our StratoCyberLab:

If you're having issues with last assignment and getting 500 Internal Server Error, your system incorrectly converted line endings in the git repository.

To fix it you must delete strato cyber lab, setup git to correct line endings and git clone it again. So, the following should work:

1. Stop your docker container for stratocyberlab by running `docker compose down`
2. Delete the folder where you cloned the repository by running `rm -r stratocyberlab`
3. Setup `git` to not to convert line endings on checkout
    1. `git config --global core.autocrlf false`
    2. `git config --global core.eol lf`
4. Clone the StratoCyberLab by running `git clone https://github.com/stratosphereips/stratocyberlab.git`
5. Build & start it again by running `docker compose up -d --build --force-recreate`

**Reply by friggingee** [reply]
Answer from @LukasForst and @HappyStoic:

For people with Windows machines running our StratoCyberLab:

If you're having issues with last assignment and getting 500 Internal Server Error, your system incorrectly converted line endings in the git repository.

To fix it you must delete strato cyber lab, setup git to correct line endings and git clone it again. So, the following should work:

1. Stop your docker container for stratocyberlab by running `docker compose down`
2. Delete the folder where you cloned the repository by running `rm -r stratocyberlab`
3. Setup `git` to not to convert line endings on checkout
   1. `git config --global core.autocrlf false`
   2. `git config --global core.eol lf`
4. Clone the StratoCyberLab by running `git clone https://github.com/stratosphereips/stratocyberlab.git`
5. Go into the folder again `cd stratocyberlab/`
6. Build & start it again by running `docker compose up -d --build --force-recreate`
